front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -782,21 +795,15 @@ class SettingsCustomizeTab extends PureComponent<
                     </LabelContent>
                   </ToggleLabel>
                 </Setting>
-              </WideSectionField>
-
-              {tenant.data.attributes.settings?.events_widget?.allowed && (
-                <Outlet
-                  id="app.containers.Admin.settings.customize.EventsWidgetSwitch"
-                  checked={this.getSetting('events_widget.enabled')}
-                  onChange={this.handleToggleEventsWidget}
-                  title={formatMessage(messages.eventsWidgetSetting)}
```

Comment:
Removed title and description prop; translations for components that render in outlets should go in the modules where the components reside.